app/assets/src/components/Samples.jsx

```diff
@@ -987,32 +1010,12 @@ class Samples extends React.Component {
       </div>
     );
 
-    const compare_button_inner = (
-      <span>
-        <span
-          className="img-container compare-container"
-          dangerouslySetInnerHTML={{
-            __html: IconComponent.compare(colors.disabledGray)
-          }}
-        />
-        <span>Compare</span>
-      </span>
-    );
-
     let compare_button = (
-      <div className="compare-area">
-        <div className="white">
-          {this.state.selectedSampleIds.length > 0 ? (
-            <a onClick={this.compareSamples} className="compare center">
-              {compare_button_inner}
-            </a>
-          ) : (
-            <a className="compare center btn-disabled">
-              {compare_button_inner}
-            </a>
-          )}
-        </div>
-      </div>
+      <CohortButton
```

Comment:
Why Cohort? 
We should give more descriptive names.

Reply:
The idea was that this is a button representing an action on a group of samples, i.e. a "cohort" -- such as comparing them in a heatmap or aggregating them in a background. Better name suggestions welcome -- what did you have in mind?

Reply:
I do not think the idea is to use the same exact button for "Comparing" and "Create Background". The icon was design for comparing only. 
I am working on organizing all our UI assets. I believe we should have generic buttons: "PrimaryButton", "SecondaryButton" that have specific designs. And for button with specific icons, we will give the name of the action "CompareButton"... Currently, I think you should a generic primary button instead of the compare.

Reply:
sounds good